File: eTeatar/eTeatar.Model/KartaDTO.cs
  using System;
using System.Collections.Generic;
using System.Text;

namespace eTeatar.Model
{
    public class KartaDTO
    {
        public int KartaId { get; set; }
        public decimal Cijena { get; set; }
        public int SjedisteId { get; set; }
        public string Red { get; set; }
        public string Kolona { get; set; }
        public int TerminId { get; set; }
        public DateTime DatumVrijeme { get; set; }
        public string NazivPredstave { get; set; }
    }
}

File: eTeatar/UI/eteatar_mobile/lib/models/karta_dto.dart
import 'package:json_annotation/json_annotation.dart';

part 'karta_dto.g.dart';

@JsonSerializable()
class KartaDTO {
  final int kartaId;
  final double cijena;
  final int sjedisteId;
  final String red;
  final String kolona;
  final int terminId;
  final DateTime datumVrijeme;
  final String nazivPredstave;
  final bool?  ukljucenaHrana;
  KartaDTO(
    this.kartaId,
    this.cijena,
    this.sjedisteId,
    this.red,
    this.kolona,
    this.terminId,
    this.datumVrijeme,
    this.nazivPredstave,
    this.ukljucenaHrana,
  );

  factory KartaDTO.fromJson(Map<String, dynamic> json) => _$KartaDTOFromJson(json);

  /// Connect the generated [_$GlumacToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$KartaDTOToJson(this);
}